www/include/configuration/configObject/service_template_model/listServiceTemplateModel.php

```diff
@@ -304,6 +327,8 @@ function setO(_i) {
 
 $tpl->assign('limit', $limit);
 $tpl->assign('searchST', $search);
+$tpl->assign("displayLockedChecked", $displayLockedChecked);
+$tpl->assign('displayLocked', _("Display locked"));
```

Comment:
Please add and translate in french this new text :
/centreon/install/smarty_translate.php
/centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

Can you add details about this new fonctionnality in the documentation too ?
/centreon/doc/en & /centroen/doc/fr